spaceship/lib/spaceship/connect_api/token.rb

```diff
@@ -22,7 +22,6 @@ class Token
       attr_reader :expiration
 
       attr_reader :key_raw
-      attr_reader :is_key_content_base64
```

Comment:
it is added by https://github.com/fastlane/fastlane/pull/20631/files#diff-ba9086d4da8fe68c8c84efaffba0f85f39d64b5a1c08b0a6db13360783e10007R25 but now no longer necessary